src/NanoVG/Internal/GLES3.chs

```diff
@@ -0,0 +1,51 @@
+module NanoVG.Internal.GLES3 where
+
+import qualified Data.Set as S
+import           Data.Word
+import           Foreign.C.Types
+
+import           NanoVG.Internal.GL3 (CreateFlags(..))
+import           NanoVG.Internal.Types
+import           NanoVG.Internal.Context
+import           NanoVG.Internal.FFIHelpers
+
+-- For now only the GL3 and GLES3 backends are supported
+#define NANOVG_GLES3
+-- We need to include this to define GLuint
+#if defined(darwin_HOST_OS)
+#include <OpenGL/gl3.h>
+#else
+#include "GL/glew.h"
+#endif
+#include "nanovg.h"
+#include "nanovg_gl.h"
+
+{#pointer *NVGcontext as Context newtype nocode#}
```

Comment:
It seems like this is defined in exactly the same way for the GL3 backend and the GLES3 backend. The same holds for `NVGcreateFlags` and `Gluint`. How about moving this to a separate module that’s shared between both?